app/assets/javascripts/dashboard/views/logged_time_view.js

```diff
@@ -9,11 +9,11 @@ import C3Chart from "react-c3js";
 
 const { Column } = Table;
 
-export default class LoggedTimeView extends React.PureComponent {
-  props: {
-    userID: ?string,
-  };
-
+export default class LoggedTimeView extends React.PureComponent<{
+  userID: ?string,
+}, {
+  timeEntries: Array<Object>,
+}> {
   state: {
```

Comment:
Good catch, a bummer, that the upgrade tool didn't take care of that. Jesus, I did it manually now ^^